src/renderer/services/db/serializers/ssz.ts

```diff
@@ -0,0 +1,18 @@
+import { CGSerialization } from '../';
+import { AnySSZType, deserialize, hashTreeRoot, serialize } from '@chainsafe/ssz';
+
+class SSZSerializer implements CGSerialization<AnySSZType> {
+    serialize(value: any, type: AnySSZType): Buffer {
+        return serialize(value, type);
+    }
+    deserialize(value: Buffer, type: AnySSZType): any {
+        return deserialize(value, type);
+    }
+    hashTreeRoot(value: any, type: AnySSZType): Buffer {
+        return hashTreeRoot(value, type);
+    }
+}
+
+const SSZ = new SSZSerializer();
+
+export default SSZ;
```

Comment:
don't use export default. I will try to update eslint ro prefer named export

Reasoning:
https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/